utils/log.sh

```diff
@@ -175,23 +175,34 @@ overwrite() {
 #
 # Arguments:
 # - $1 : command (eg. "docker service rm elastic-search")
-# - $2 : error message (eg. "Failed to remove elastic-search service")
+# - $2 : throw or catch (eg. "throw", "catch")
+# - $3 : error message (eg. "Failed to remove elastic-search service")
 try() {
     local -r COMMAND=${1:?"FATAL: function 'try' is missing a parameter"}
-    local -r ERROR_MESSAGE=${2:?"FATAL: function 'try' is missing a parameter"}
+    local -r THROW=${2:-"throw"}
```

Comment:
Let's deal with this as a boolean : SHOULD_THROW.

Reply:
cool done in [201d509](https://github.com/jembi/platform/pull/188/commits/201d50983863521fe4ed85a031505b5d1e4a8e87)